### PR TITLE
clean up disabled tests

### LINKED ISSUE #220: check disabled tests
why are they disabled, whether we can make them more enabled

### PR DESCRIPTION
**Pull request by bbjubjub2494**
Fixes #220

### CODE REVIEW

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/model/UniqueDatabaseManagementTest.kt [hunk 1]
```diff
@@ -70,16 +68,4 @@ class UniqueDatabaseManagementTest {
             equalTo(oDbman.getCategories().map { cat -> cat.id })
         )
     }
-
-    @Ignore("functionality removed for the time being")
-    @Test
-    fun offlineDatabaseThrowsIfNotDownloaded() = runBlocking {
```
Comment (bbjubjub2494) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/model/UniqueDatabaseManagementTest.kt:76]: - If there's no functionality to test, yeet
- This doesn't call into the DUT, instead it goes to an internal constructor of a different class. Irrelevant, therefore yeet

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt [hunk 2]
```diff
@@ -167,22 +165,6 @@ class MetadataEditingFragmentTest {
         )
     }
 
-    @Ignore("will be fixed later")
-    @Test
-    fun clickOnInfoButtonWorks() {
```
Comment (bbjubjub2494) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt:172]: There is no info button on this view anyway.

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/EvaluationResultFragmentTest.kt [hunk 3]
```diff
@@ -53,7 +52,6 @@ class EvaluationResultFragmentTest {
 
 
     @Test
-    @Ignore("I don't know the library well enough to test its content yet")
     fun rightStuffIsDisplayedOnGraph() {
```
Comment (bbjubjub2494) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/EvaluationResultFragmentTest.kt]: Do you now @mlenw ?

File: app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/SelectPictureFragment.kt [hunk 4]
```diff
@@ -39,7 +39,7 @@ class SelectPictureFragment : Fragment() {
     private val binding get() = _binding!!
 
     private val args: SelectPictureFragmentArgs by navArgs()
-
+
```
Comment (bbjubjub2494) [inline app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/SelectPictureFragment.kt]:
```suggestion

```

note to self

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt (hunk 2 above)
Comment (bbjubjub2494) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt:172]: #233

**Review by NicolasVial — APPROVED** [review]
Looks good. With Martin0s answer we should be good to merge this. And I confirm that the test clickOnInfoButtonWorks is fixed by my PR.

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/TestUtils.kt [hunk 5]
```diff
@@ -112,4 +114,23 @@ object TestUtils {
         { "There has to be a dataset with $N categories in $dbMgt" }
         return maybeDataset
     }
+
+    // per https://stackoverflow.com/a/60032604
+    fun setTextInSearchView(value: String): ViewAction {
```
Comment (martin-lenweiter) [inline app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/TestUtils.kt:119]: nice

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/model/UniqueDatabaseManagementTest.kt (hunk 1 above)
Comment (martin-lenweiter) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/model/UniqueDatabaseManagementTest.kt:76]: ok

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt (hunk 2 above)
Comment (martin-lenweiter) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt:172]: ok

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/TakePictureActivityTest.kt [hunk 6]
```diff
@@ -206,34 +206,6 @@ class TakePictureActivityTest {
         onView(withId(R.id.selectCategoryButton)).check(matches(hasTextColor(R.color.black)))
     }
 
-
-    /*
-    TODO: convert to fragment
-
-    @Test
-    fun receiveIntentFromCamera() {
```
[the remark this answers is not in the document]
Comment (martin-lenweiter) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/TakePictureActivityTest.kt:214]: ok

File: app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/EvaluationResultFragmentTest.kt (hunk 3 above)
Comment (martin-lenweiter) [reply app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/EvaluationResultFragmentTest.kt]: unfortunately not.

**Review by martin-lenweiter — APPROVED** [review]
All good. We can also ditch the mpandroidchart test, as I don't know how to do it, and the graph does the basic thing it should do, seen through manual testing (not ideal but that's what we got).

**Pull request merged**